buildSrc/src/main/kotlin/com/virtuslab/gitmachete/buildsrc/UiTests.kt

```diff
@@ -13,6 +13,9 @@ fun Project.configureUiTests() {
   val isCI: Boolean by rootProject.extra
   val intellijVersions: IntellijVersions by rootProject.extra
 
+  tasks["compileUiTestScala"].finalizedBy("classpathIndexCleanup")
```

Comment:
So that sh*t like that doesn't show up:

```
>  Task :classpathIndexCleanup
Execution optimizations have been disabled for task ':classpathIndexCleanup' to ensure correctness due to the following reasons:
  - Gradle detected a problem with the following location: '/Users/pawel_pawellipski/git-machete-intellij-plugin/build/classes/scala/uiTest/classpath.index'. Reason: Task ':classpathIndexCleanup' uses this output of task ':compileUiTestScala' without declaring an explicit or implicit dependency. This can lead to incorrect results being produced, depending on what order the tasks are executed. Please refer to https://docs.gradle.org/7.6/userguide/validation_problems.html#implicit_dependency for more details about this problem.
  - Gradle detected a problem with the following location: '/Users/pawel_pawellipski/git-machete-intellij-plugin/build/generated/sources/annotationProcessor/scala/uiTest/classpath.index'. Reason: Task ':classpathIndexCleanup' uses this output of task ':compileUiTestScala' without declaring an explicit or implicit dependency. This can lead to incorrect results being produced, depending on what order the tasks are executed. Please refer to https://docs.gradle.org/7.6/userguide/validation_problems.html#implicit_dependency for more details about this problem.

> Task :processUiTestResources
Execution optimizations have been disabled for task ':processUiTestResources' to ensure correctness due to the following reasons:
  - Gradle detected a problem with the following location: '/Users/pawel_pawellipski/git-machete-intellij-plugin/build/resources/uiTest'. Reason: Task ':classpathIndexCleanup' uses this output of task ':processUiTestResources' without declaring an explicit or implicit dependency. This can lead to incorrect results being produced, depending on what order the tasks are executed. Please refer to https://docs.gradle.org/7.6/userguide/validation_problems.html#implicit_dependency for more details about this problem.
```